### ISSUE TITLE
Profile feature

### ISSUE THREAD

**Issue by heron2014** [DRDD2016/native#143]
- [x] resuse user state to display data in Profile page
- [x] edit firstname and surname for now
- [x] send request to server: PATCH /users/:user_id
- [x] clean up user state and profile state

**Comment by heron2014** [comment]
Logout interesting behaviour: 
After removing token from AsyncStorage I used: 

```js
resetToStack('auth')
```

and I got redirected to this:

<img width="283" alt="screen shot 2017-02-13 at 22 25 35" src="https://cloud.githubusercontent.com/assets/8476884/22906286/b74b8a16-f23b-11e6-84ba-d063b1e24da7.png">

Solution to solve this problem was: 

In our ```index.js```

```js
class App extends Component {

  render () {
    return (
      <Provider store={ store }>
        <NavigationProvider context={ navigationContext }>
          <StackNavigation id="root" navigatorUID="root" initialRoute={ Router.getRoute('auth') } />
        </NavigationProvider>
      </Provider>
    );
  }
}
```
I added ID to navigator: ```root```

And after removing token from AsyncStorage I used slightly different approach:

```js
  const rootNavigator = navigation.getNavigator('root');
  rootNavigator.replace('auth');
```

and this worked.

**Comment by heron2014** [comment]
Resolved by #146
